### ISSUE TITLE
Augmenting validation set shouldn't be mandatory

### ISSUE THREAD

**Issue by maxence-ai** [karolzak/keras-unet#26]
In [utils.py](https://github.com/karolzak/keras-unet/blob/f8f9705477aa25c356e4612d0be403b7aedd2647/keras_unet/utils.py#L68), the validation set is augmented. AFAIK the validation set should be fixed and be as close as real world data. In this case it is manually changed and won't be the same across two epochs.

Please let me know if I'm missing something. I'd go with `shuffle=False`, because if your dataset isn't a multiple of the batch size the dataset won't be the all the time the same and `augment=False` by default for the aformentioned reasons.

**Comment by karolzak** [comment]
@maxpv thanks for picking that up!  
You're absolutely correct! And I swear to god it used to be no augment and no shuffle for validation when I was writing it and using it 😆. I guess when I was putting this stuff into keras-unet package and doing some refactoring I must have mistakenly copy-pasted something wrong. That's what you get for not writting tests...

**Comment by maxence-ai** [comment]
No problem, actually I doubled checked if anyone is actually doing augmentation on the validation set and found [this interesting thread](https://stats.stackexchange.com/a/320967).

**Comment by karolzak** [comment]
> No problem, actually I doubled checked if anyone is actually doing augmentation on the validation set and found [this interesting thread](https://stats.stackexchange.com/a/320967).

Interesting indeed. Some good points there, although for me the main purpose of augmentation is to reduce overfitting and having that in mind (as you said before) you don't want to run augmentation on your val/test datasets - you rather have it fixed and always the same

**Comment by maxence-ai** [comment]
Yes we agree that they are edge cases.
